doc/users/next_whats_new/subfigures_change_order.rst

```diff
@@ -1,3 +1,10 @@
+Subfigures no longer provisional
+--------------------------------
+
+The API on `.Figure.subfigures` and `.SubFigure` are now considered stable and
+will change under Matplotlib's normal deprecation process.
```

Comment:
```suggestion
The API on `.Figure.subfigures` and `.SubFigure` are now considered stable.
```
I would leave that out. It confuses more than it helps. What people read from this is "The API ... will change". What we want to communicate is that the API does not change (and if there should a need for change araise, it will be announced beforehand and include a migration path). But that's just the standard way of working in the library and does not need explicit mentioning.

Reply:
I copied from https://matplotlib.org/stable/users/prev_whats_new/whats_new_3.7.0.html#subplot-mosaic-no-longer-provisional but sure.